pkcs11-env.h

```diff
@@ -29,6 +29,24 @@
 #define NULL_PTR 0
 #endif
 
+#ifdef PKCS11_LONGTYPE
+ #include <stdint.h>
+ #if PKCS11_LONGTYPE==32
+ typedef uint32_t PKCS11_ULONG_TYPE;
+ typedef int32_t PKCS11_LONG_TYPE;
+ #elif PKCS11_LONGTYPE==64
+ typedef uint64_t PKCS11_ULONG_TYPE;
+ typedef int64_t PKCS11_LONG_TYPE;
+ #else
+ #warning "Invalid value for PKCS11_LONGTYPE, defaulting to long"
```

Comment:
Let's make this an `#error` to have a better chance of catching any unwanted build combinations.